sync.go

```diff
@@ -125,37 +145,29 @@ func truncate(rev string) string {
 	return rev
 }
 
-func syncPkg(ch chan<- string, importPath, expectedRevision string) {
+func syncPkg(ch chan<- string, importPath, expectedRevision, getOutput string, getErr error) {
 	var importDir = filepath.Join(gopath(), "src", importPath)
 	var status bytes.Buffer
 	defer func() { ch <- status.String() }()
 
 	// Try to find the repo.
-	var getOutput []byte
 	var repo, err = fastRepoRoot(importPath)
 	if err != nil {
-		// go get it in case it doesn't exist. (no-op if it does exist)
-		// (ignore failures due to "no buildable files" or build errors in the package.)
-		var getErr error
-		getOutput, getErr = run("go", "get", "-v", "-d", importPath)
-		repo, err = fastRepoRoot(importPath)
-		if err != nil {
-			var getStatus = "(success)"
-			if getErr != nil {
-				getStatus = string(getOutput) + getErr.Error()
-			}
-			perror(fmt.Errorf(`failed to get: %s
+		var getStatus = "(success)"
+		if getErr != nil {
+			getStatus = string(getOutput) + getErr.Error()
+		}
+		perror(fmt.Errorf(`failed to get: %s
 
 > go get -v -d %s
 %s
 
 > import %s
 %s`, importPath, importPath, getStatus, importPath, err))
-		}
 	}
 
 	var maybeGot = ""
-	if bytes.Contains(getOutput, []byte("(download)")) {
+	if strings.Contains(getOutput, importPath+" "+"(download)") {
```

Comment:
I meant why not `importPath + " (download)"` instead of `importPath + " " + "(download)"`